src/main/java/gregtech/api/model/modelfactories/BakedModelHandler.java

```diff
@@ -81,6 +79,9 @@ public void onModelsBake(ModelBakeEvent event) {
             ModelBuiltInRenderer bakedModel = new ModelBuiltInRenderer(tuple.getSecond());
             event.getModelRegistry().putObject(resourceLocation, bakedModel);
         }
+        event.getModelRegistry().putObject(BlockOre.MODEL_LOCATION, OreBakedModel.INSTANCE);
+        event.getModelRegistry().putObject(BlockCompressed.MODEL_LOCATION, CompressedBlockBakedModel.INSTANCE);
+        event.getModelRegistry().putObject(BlockFrame.MODEL_LOCATION, FrameBakedModel.INSTANCE);
```

Comment:
This was placed here for the time being, didn't have time to redo this part, but this code should be in a more visible place within the codebase and should be where all models are handled.